portfolio/src/main/webapp/index.html

```diff
@@ -122,6 +128,9 @@ <h3 id="contact_me">Contact Me</h3>
             <option value="both cats and dogs">both!</option>
           </select>
           <br>
+          <label for="comment-image">Upload image</label>
```

Comment:
On the deployed version of your site, comment submission fails when you try to upload an image (probably because of the API permissions for the GCP project).  Right now it's posting a stack track of the error, but potentially you could catch the error more gracefully and just show an "invalid" message like "Image upload not supported right now" or something.